docs/source/pipeline_canfar.md

```diff
@@ -0,0 +1,95 @@
+patch="P7"
```

Comment:
Specific step-by-step commands (pipeline) to produce shape catalogue for one patch on the canfar science portal.